generators/base-generator.js

```diff
@@ -74,12 +74,12 @@ module.exports = class extends Generator {
     }
 
     _formatCodeMaven() {
-        const command = this._isWin() ? 'mvnw.bat' : './mvnw';
+        const command = this._isWin() ? 'mvnw' : './mvnw';
```

Comment:
On Windows OS shouldn't we use `mvnw.bat`?

Reply:
I'm using windows Os and only normal command is working , i.e with out bat.

Ideally with bat also should work but couldn't figure it out the reason